vermouth/dssp/dssp.py

```diff
@@ -461,13 +468,14 @@ def convert_dssp_annotation_to_martini(
 class AnnotateDSSP(Processor):
     name = 'AnnotateDSSP'
 
-    def __init__(self, executable='dssp', savedir=None):
+    def __init__(self, executable='dssp', savedir=None, version='3.0.0'):
         super().__init__()
         self.executable = executable
         self.savedir = savedir
+        self.version = version
 
     def run_molecule(self, molecule):
-        annotate_dssp(molecule, self.executable, self.savedir)
+        annotate_dssp(molecule, self.executable, self.savedir, self.version)
```

Comment:
this line is incorrect, because self.versoin is not the 4th positional argument. This should be replaced by:

```
annotate_dssp(molecule, self.executable, self.savedir, 'secstruct', self.version)
```

Otherwise the dssp string doesn't get annotated correctly, which causes the CI fail. Ohh boy am I happy we have those tests.

Reply:
I'm happy you found this :)

Reply:
Thank you! Fixed in commit 8793110397c4f5bbc84e5dbab7a447a7be58abca.